packages/twenty-server/src/workspace/workspace-sync-metadata/standard-objects/activity-target.object-metadata.ts

```diff
@@ -57,4 +59,13 @@ export class ActivityTargetObjectMetadata extends BaseObjectMetadata {
   })
   @IsNullable()
   opportunity: OpportunityObjectMetadata;
+
+  @ComputedRelationFieldMetadata((oppositeObjectMetadata) => ({
```

Comment:
I feel the naming is not clear enough. Could we name it:
@RelationToAnyCustomObjectMetadata

Reply:
I agree, except if this can be extended to other use cases, then Computed is more generic and clear about what it's doing "it computes a RelationField based of the current object and opposite object)

Reply:
I think the name `@RelationToAnyCustomObjectMetadata` is not clear enough, as in this case we're creating a computed `FieldMetadata` based on the opposite objectMetadata.
I like `ComputedRelationFieldMetadata` because it says, it's computed, it's a type RELATION and it's a FieldMetadata, maybe we can try to find something else ?

Reply:
"Computed" will be used for ComputedFields. DynamicRelationFieldMetadata?

Reply:
Good to me !